cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -16,62 +16,122 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 == New and changed features
 
-=== Supported binding methods
-* Open vSwitch (OVS) is no longer recommended and should not be used
-in {CNVProductName} 2.0.
-* For the default Pod network, `masquerade` is the only recommended binding
-method. It is not supported for non-default networks.
-* For secondary networks, use the `bridge` binding method.
-
 === Web console improvements
-* You can now view all services associated with a virtual machine in the
-*Virtual Machine Details* screen.
-
 
-== Resolved issues
+* The {product-title} dashboard captures high-level information about clusters.
+From the OpenShift web console, access the dashboard by clicking
```

Comment:
s/OpenShift web console/{product-title} web console
We use product title on first instance for web console.